src/content/docs/apm/agents/ruby-agent/getting-started/ruby-agent-requirements-supported-frameworks.mdx

```diff
@@ -16,70 +16,54 @@ redirects:
   - /docs/ruby/rubinius
   - /docs/agents/ruby-agent/troubleshooting/new-relic-puma
 ---
-
```

Comment:
Do you know if these whitespace changes impact the content's readability?

It looks like the Gatsby build is still in progress, so I can't check this myself yet.